engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
+## Leads Meeting
 
+Same as something called SoS before - a weekly meeting on Tuesdays 14:30 of CTO
+\w all team leads to catch up on ongoing progress.
 
-### Planning
+Each team lead will prepare:
 
-Planning meetings
+* Review on what has been worked last week
+* What are the things we are looking forward to? (re-prioritize things)
+* Pitch new ideas from our workflow
 
-  + It is really a planing DAY, not just a meeting:
+Then grooming company’s backlog/epics from <> is conducted.
 
-    * There will be a 1 hour long kickoff meeting at 10:00 (first team) and
-      11:00 (second team)
+## Team Meeting
 
-    * The rest of the day, the team will think about proposed sprint plan and
-      decide if it is doable.
+A Bi-weekly meeting of the CTO with the team lead and optionally with the full
+team (to be decided by the lead), in order to comment the status of the
+projects. Scheduled with each team individually by CTO.
 
-         - leads review backlog before the planning
-         - read backlog
-         - agree on scope of sprint
-         - estimate
+## Standup
 
-    * There will be a close meeting at 17:30 (first) and 18:00 (second) to confirm
-      the sprint plan.
+All standups are happening online and asynchronously, per-team in appropriate
+`su-*` Slack channels.
 
-  + All OKR objectives for each team member will be covered in the planning.
+## Demo
 
+Demo meetings are monthly event.
 
-### Deployment
+* Idea: let know everybody what's going on (on quite technical level) that's is relevant for other or affect others releases/deployments.
+* Product does not need to be deployed, but the demo should be as real as possible.
+* Should be concise, respecting other people's time.
+* Fix max time **per-team: 15min (including questions)**. Agree between teams in advance, if more time needed.
 
-  + Deploying may not make sense for some teams, i. e. Data Science and Devops.
+## Keeping track of things
 
-  + Tools and Data Retrieval will deploy on Mondays, after the end of an sprint,
-    as they don't have the planing meeting until Tuesday.
+Github Projects board will used as a Kanban board.
 
-### Demo
+Teams managing issues across Github Org borders (Applications, Language Analysis) will use the “add card”  trick, described in dear-github/dear-github#209 to  add issues from different org  to the team’s board.
 
-Demo meetings
+Issues repositories:
 
-  + Idea: let know everybody what's going on (on quite technical level).
+* [src-d/issues-general](https://github.com/src-d/issues-general) - should be used more, as place for feedback/requests (i.e website, etc)
+* [src-d/issues-infrastructure](https://github.com/src-d/issues-infrastructure) - issues/requests to Infrastructure team must be created here.
 
-  + At the end of each sprint: Fridays 13:30.
+# Definitions
 
-  + Product does not need to be deployed, but the demo should be as real as
-    possible.
+Below the definitions of Teams Kanban columns are given.
```

Comment:
"Below, we'll define the meaning of each Kanban column" would be better, I think.